src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private double threshold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        threshold = DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (containsKey(key)) {
+            Node<K, V> currentNode = getNode(key);
```

Comment:
You call `getNode` inside `containsKey`. It means you call it twice here. It's not good for the performance.